lib/animated_orbit_menu.dart

```diff
@@ -1,36 +1,42 @@
 import 'package:flutter/material.dart';
+import 'package:orbitmenu/orbitmenu.dart';
 import 'orbit_menu_config.dart';
-import 'OrbitMenu.dart';
 import 'orbit_menu_animation_type.dart';
 import 'dart:math';
 
 class AnimatedOrbitMenu extends StatefulWidget {
   final bool animate;
   final OrbitMenuConfig config;
 
-  AnimatedOrbitMenu({Key? key, required this.animate, required this.config}) : super(key: key);
+  const AnimatedOrbitMenu(
+      {Key? key, required this.animate, required this.config})
+      : super(key: key);
 
   @override
   _AnimatedOrbitMenuState createState() => _AnimatedOrbitMenuState();
 }
 
-class _AnimatedOrbitMenuState extends State<AnimatedOrbitMenu> with SingleTickerProviderStateMixin {
+class _AnimatedOrbitMenuState extends State<AnimatedOrbitMenu>
+    with SingleTickerProviderStateMixin {
   late AnimationController _controller;
   late Animation<double> _animation;
 
   @override
   void initState() {
     super.initState();
     if (widget.animate) {
-      _controller = AnimationController(vsync: this, duration: widget.config.animationDuration);
+      _controller = AnimationController(
+          vsync: this, duration: widget.config.animationDuration);
```

Comment:
In the Pass static analysis, they mentioned I needed to format it in Dart style or something like that. But let's try and see what happens; if it doesn't work, we'll revert it back to how it was. For now, I'll approve it.